### PR TITLE
feat: fontInput updates

### PR DESCRIPTION
**Pull request by anxobotana**

### CODE REVIEW

File: packages/sidebar-settings/src/blocks/fontInput.ts
Comment (SamuelAlev) [inline packages/sidebar-settings/src/blocks/fontInput.ts:1]: Please add a changeset `pnpm changeset`

**Review by mike85 — APPROVED** [review]
LGTM

File: packages/sidebar-settings/src/blocks/fontInput.ts [hunk 1]
```diff
@@ -249,4 +382,8 @@ export type FontInputBlock<AppBridge> = {
      * @default "Lorem Ipsum"
      */
     placeholder?: string;
+    predefinedChoiceValues?: {
```
[the remark this answers is not in the document]
Comment (anxobotana) [reply packages/sidebar-settings/src/blocks/fontInput.ts:389]: Yes and no. What happens with fontInput block is that contains multiple blocks inside, `defaultValue` sets the defaults for each of those internal blocks. What this `predefinedChoiceValues` does is allowing to modify some of the visual elements. For example, for padding we'll have in some cases: Auto/S/M/L and in other cases None/S/M/L. We use the same type, but we set the label `Auto` from here to what ever we need, in this case `None`.
Also it allows to show in the preview the actual padding and border radius.

File: packages/sidebar-settings/src/blocks/fontInput.ts [hunk 2]
```diff
@@ -249,4 +382,8 @@ export type FontInputBlock<AppBridge> = {
...
+        [choice: string]: { [option: string]: string } | string;
+    };
+    stateTabsEnabled?: 'none' | 'all';
```
[the remark this answers is not in the document]
Comment (anxobotana) [reply packages/sidebar-settings/src/blocks/fontInput.ts:396]: jsdocs added

[the remark this answers is not in the document]
Comment (anxobotana) [reply packages/sidebar-settings/src/blocks/fontInput.ts]: Replace this with an option per state tab.

File: packages/sidebar-settings/src/blocks/fontInput.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (anxobotana) [reply packages/sidebar-settings/src/blocks/fontInput.ts]: After talking to @mike85 and @SamuelAlev we all agreed on `presetChoices` for the naming for this.

File: packages/sidebar-settings/src/blocks/fontInput.ts [hunk 3]
```diff
@@ -249,4 +384,19 @@ export type FontInputBlock<AppBridge> = {
...
      */
     placeholder?: string;
+
+    /**
+     * Choice values presets:
+     *    paddingSizeChoiceValues: choices values for padding sizes choices
+     *    borderRadiusChoiceValues: choices values for border radius choices
+     *    aligmentChoiceValues: text alignment choices
+     */
+    choicePresets?: FontInputChoiceValues;
```
Comment (SamuelAlev) [inline packages/sidebar-settings/src/blocks/fontInput.ts:394]:
```suggestion
    choicePresets?: FontInputChoicePresetValues;
```

**Review by SamuelAlev — APPROVED** [review]
Left small comment but nothing blocking

File: packages/sidebar-settings/src/blocks/fontInput.ts (hunk 3 above)
Comment (anxobotana) [reply packages/sidebar-settings/src/blocks/fontInput.ts]: changed!

**Pull request merged**